### ISSUE TITLE
Change Bundle Qt backend to PyQt5 

### ISSUE THREAD

**Issue by Czaki** [napari/napari#4514]
## 🚀 Feature

Change backend of the bundled app to PyQt5 

## Motivation

PyQt5 in version 5.15.4 is released in conda-forge https://anaconda.org/conda-forge/pyqt 
PySide2 is still available in version 5.13.2 which uses Qt in version 5.12.9. 
Switching to PyQt5 should reduce the problems with lack of features in old  Qt bindings releases (for example QComboBox.addItems for new version accept iterators (ex. `dict.keys()`), but PySide2 from the conda forge do not accept.

**Comment by psobolewskiPhD** [comment]
Yup!
Includes macOS arm64 binaries too!
`conda create -y -n napari-env python=3.9 napari`
JustWorks!

**Comment by jaimergp** [comment]
This will happen automatically on arm64 (because PySide2 is not there yet). We also use conda-forge on this platform [now](https://github.com/napari/napari/pull/4404/commits/c67551430e832279df10d04ef5d1324256e46754), too. 

If I remember correctly, PySide is the default option due to the licensing. It's a pity that conda-forge does not have 5.15 yet, but there are [ongoing attempts](https://github.com/conda-forge/pyside2-feedstock/pulls))!

**Comment by Czaki** [comment]
> If I remember correctly, PySide is the default option due to the licensing

Yes but currently napari does not ship Qt bindings in the bundle but download it from conda-forge so there is no problem with license.

**Comment by jaimergp** [comment]
I am not sure that's how GPL works. The bindings are indeed shipped in the bundled installer (as a conda package downloaded from conda-forge, that part is right).

**Comment by Czaki** [comment]
I have an additional argument for changing the backend. It looks like most of the napari developers are using PyQt5 as the backend. 100% testing is hard. For GUI it is even harder. But such bugs could be found by daily using `napari` by contributors. 

But if most of the contributors are using PyQt5 then PySide2 bugs will be not found. And then casual users, who are not familiar with bug reporting, use bundled versions and meet bugs but do not inform us about the existing problems. 

Bug fixed in #4581 exists since 0.4.15 release and I worry this is not the only one.

**Comment by goanpeca** [comment]
>Yes but currently napari does not ship Qt bindings in the bundle but download it from conda-forge so there is no problem with license.

Yes it does.

>But if most of the contributors are using PyQt5 then PySide2 bugs will be not found. 

Then we should probably (napari team, contributors) start using pyside2.
